modular/gater/admin_handler.go

```diff
@@ -488,6 +492,62 @@ func (g *GateModular) replicateHandler(w http.ResponseWriter, r *http.Request) {
 	log.CtxDebug(reqCtx.Context(), "succeed to replicate piece")
 }
 
+func (g *GateModular) checkReplicatePermission(ctx context.Context, receiveTask gfsptask.GfSpReceivePieceTask, signatureAddr string) error {
+	objectInfo, err := g.baseApp.Consensus().QueryObjectInfo(ctx, receiveTask.ObjectInfo.BucketName, receiveTask.ObjectInfo.ObjectName)
```

Comment:
can use GfSpClient().GetObjectByID(objectId) and GfSpClient().GetGlobalVirtualGroup(bucketId, lvgId) to get GVG.